### PR TITLE
feat(tools): automatically lowercase directories

### PR DESCRIPTION
**Pull request by a2937**
Checklist:

- [X] I have read and followed the [contribution guidelines](https://contribute.freecodecamp.org).
- [X] I have read and followed the [how to open a pull request guide](https://contribute.freecodecamp.org/#/how-to-open-a-pull-request).
- [X] My pull request targets the `main` branch of freeCodeCamp.
- [X] I have tested these changes either locally on my machine, or GitPod.

Closes #XXXXX

I wanted to use the lowercase directories plugin but because I forgot that the total number of directories underneath the relevant folders isn't known and can change over time, it was best to use a shell command.

### CODE REVIEW

**Review by naomi-lgbt — CHANGES_REQUESTED** [review]
Hmmm, I think it might be better to update the plugin to be recursive?

File: .github/workflows/crowdin-download.client-ui.yml [hunk 1]
```diff
@@ -328,6 +328,13 @@ jobs:
           sudo chown -R $(whoami): client/i18n/locales
           npx --yes prettier --write client/i18n/locales/**/*.json
 
+      ###### Lowercase directory names #####
+
```
Comment (Nightrider1983) [inline .github/workflows/crowdin-download.client-ui.yml:332]: 000/$/

**Comment by ojeytonwilliams** [comment]
Hi @a2937 are you still working on this? If you are, could you have a look at the merge conflict?

**Comment by a2937** [comment]
I will work on getting this finished as soon as I an able to though.

**Comment by ojeytonwilliams** [comment]
Awesome, thanks @a2937. We just weren't sure if you were still working on this. There's no rush, though, please take whatever time you need.

**Review by naomi-lgbt — APPROVED** [review]

**Review by Sembauke — APPROVED** [review]

**Pull request merged**